### PR TITLE
docs(ingress): Add Azure Application Gateway ingress demo

### PR DESCRIPTION
**Pull request by nshankar13**
Add demo for ingress with Azure Application Gateway

Signed-off-by: nshankar13 <nshankar@microsoft.com>

**Affected area**:

- New Functionality      [ ]
- Documentation          [X]
- Install                [ ]
- Control Plane          [ ]
- CLI Tool               [ ]
- Certificate Management [ ]
- Networking             [ ]
- Metrics                [ ]
- SMI Policy             [ ]
- Security               [ ]
- Tests                  [ ]
- CI System              [ ]
- Demo                   [ ]
- Performance            [ ]
- Other                  [ ]

Please answer the following questions with yes/no.

- Does this change contain code from or inspired by another project? If so, did you notify the maintainers and provide attribution?
No

### CODE REVIEW

File: docs/content/docs/tasks_usage/traffic_management/ingress_with_Azure_Application_Gateway.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
+---
```
Comment (shashankram) [inline docs/content/docs/tasks_usage/traffic_management/ingress_with_Azure_Application_Gateway.md:1]: Let's move this to `traffic_management/demos`, and rename the file to `azure_application_gateway_ingress_demo.md`

File: docs/content/docs/tasks_usage/traffic_management/ingress_with_Azure_Application_Gateway.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+title: "Ingress with Azure Application Gateway Demo"
+description: "Exposing services outside the cluster using Azure Application Gateway Ingress Controller"
+type: docs
+aliases: ["ingress_with_Azure_Application_Gateway.md"]
```
Comment (shashankram) [inline docs/content/docs/tasks_usage/traffic_management/ingress_with_Azure_Application_Gateway.md:5]: Alias too when filename is changed.

File: docs/content/docs/tasks_usage/traffic_management/ingress_with_Azure_Application_Gateway.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+
+    ```console
+    $ kubectl get pods
+    NAME                             READY   STATUS    RESTARTS   AGE
+    ingress-azure-5cdf9b7586-z66m9   1/1     Running   0          96m
+    mic-67bb68db85-g6v74             1/1     Running   0          102m
+    mic-67bb68db85-gk2wm             1/1     Running   0          102m
+    nmi-4dtzx                        1/1     Running   0          102m
+    nmi-d97cn                        1/1     Running   0          102m
+    nmi-wrrh7                        1/1     Running   0          102m
```
Comment (shashankram) [inline docs/content/docs/tasks_usage/traffic_management/ingress_with_Azure_Application_Gateway.md:28]:
```suggestion
    ingress-azure-5cdf9b7586-z66m9   1/1     Running   0          96m
```

**Review by shashankram — COMMENTED** [review]
Looks good, few comments.

**Review by shashankram — APPROVED** [review]

**Review by draychev — APPROVED** [review]

**Pull request merged**